services/order.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket.get("movie_session"),
+                order=order,
+                row=ticket.get("row"),
+                seat=ticket.get("seat")
+            )
```

Comment:
Consider validating the ticket data before creating `Ticket` instances. Ensure that `movie_session`, `row`, and `seat` are valid and exist in the database to maintain data integrity.